include/capo/pcm.hpp

```diff
@@ -28,7 +31,8 @@ struct PCM {
 	std::vector<Sample> samples;
 	utils::Size size{};
 
-	static Result<PCM> fromFile(std::string const& wavPath);
-	static Result<PCM> fromMemory(std::span<std::byte const> wavBytes);
+	static Result<PCM> fromFile(std::string const& path, FileFormat format);
```

Comment:
Defaulting the second param to unknown would also allow a "deduction" feature (from extension).